### PR TITLE
XFA - Textfield must a white background by default

### PR DESCRIPTION
**Pull request by calixteman**

### CODE REVIEW

File: src/display/xfa_layer.js [hunk 1]
```diff
@@ -101,6 +102,7 @@ class XfaLayer {
 
     // Set the value after the others to be sure overwrite
     // any other values.
+    console.log("COUCUO", attributes.dataId);
```
Comment (marco-c) [inline src/display/xfa_layer.js:105]: I guess these console.log should be removed :P

File: src/core/xfa/bind.js [hunk 2]
```diff
@@ -86,7 +86,6 @@ class Binder {
     // Here we make the link between form node and
     // data node (through $data property): we'll use it
     // to save form data.
-
```
Comment (marco-c) [inline src/core/xfa/bind.js:89]: Nit: unrelated newline removal

**Comment by marco-c** [comment]
Can you run the reftests?

**Comment by calixteman** [comment]
There is an error in `firefox-xfa_imm5257e-page3`: the textedit is drawn after the string so it's fine when the background is transparent but it isn't when it's white.
Maybe something is wrong with the textedit size.

**Pull request closed without merging**